xml/System.IdentityModel.Selectors/SecurityTokenResolver.xml

```diff
@@ -1,10 +1,10 @@
 <Type Name="SecurityTokenResolver" FullName="System.IdentityModel.Selectors.SecurityTokenResolver">
-  <TypeSignature Language="C#" Value="public abstract class SecurityTokenResolver" FrameworkAlternate="net-8.0;netframework-3.0;netframework-3.5;netframework-4.0;netstandard-2.0" />
-  <TypeSignature Language="ILAsm" Value=".class public auto ansi abstract beforefieldinit SecurityTokenResolver extends System.Object" FrameworkAlternate="net-8.0;netframework-3.0;netframework-3.5;netframework-4.0;netstandard-2.0" />
+  <TypeSignature Language="C#" Value="public abstract class SecurityTokenResolver" FrameworkAlternate="net-8.0;netframework-3.0;netframework-3.5;netframework-4.0" />
+  <TypeSignature Language="ILAsm" Value=".class public auto ansi abstract beforefieldinit SecurityTokenResolver extends System.Object" FrameworkAlternate="net-8.0;netframework-3.0;netframework-3.5;netframework-4.0" />
   <TypeSignature Language="DocId" Value="T:System.IdentityModel.Selectors.SecurityTokenResolver" />
-  <TypeSignature Language="VB.NET" Value="Public MustInherit Class SecurityTokenResolver" FrameworkAlternate="net-8.0;netframework-3.0;netframework-3.5;netframework-4.0;netstandard-2.0" />
```

Comment:
What does it mean for netstandard-2.0 to be removed here? SecurityTokenResolver is available for netstandard2.0 from System.ServiceModel.Primitives in 4.10.3

Reply:
The package indexer tool only considers the latest stable (and latest preview version, if it's newer), which for System.ServiceModel.Primitives is 8.0.0. Is there a need to index 4.10.3 as well?

Reply:
For now there is. We've extended support because we removed netstandard2.0 support from 8.0.0 and turns out it was more important than we were told so we're bringing it back. For now the 4.10.3 package is what you use if you need netstandard2.0 support.

Reply:
Okay, I'll special case it.

Reply:
@mconnew I'm having a hard time ingesting the System.ServiceModel.Primitives 4.10.3 package for .NET Standard. I tried with both the lib and ref assemblies, and both times got:

##[error]mdoc: Mono.Documentation.MDocAssemblyException: Error caching System.ServiceModel, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089 from D:\a\_work\1\s\source_repo\dotnet\netstandard-2.0\System.ServiceModel.dll ---> Mono.Documentation.MDocException: Failed to resolve type 'System.ServiceModel.BasicHttpBinding' ---> Mono.Cecil.AssemblyResolutionException: Failed to resolve assembly: 'System.ServiceModel.Http, Version=4.10.3.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
   at Mono.Documentation.Updater.Frameworks.MDocBaseResolver.Resolve(AssemblyNameReference name, ReaderParameters parameters, IEnumerable`1 filesToIgnore)

Should I also ingest this package for the documentation? https://www.nuget.org/packages/System.ServiceModel.Http/4.10.3

Reply:
Update: I added System.ServiceModel.Http, but now getting an error for a dependency on System.ServiceModel.Duplex:

##[error]mdoc: Mono.Documentation.MDocAssemblyException: Error caching System.ServiceModel, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089 from D:\a\_work\1\s\source_repo\dotnet\netstandard-2.0\System.ServiceModel.dll ---> Mono.Documentation.MDocException: Failed to resolve type 'System.ServiceModel.CallbackBehaviorAttribute' ---> Mono.Cecil.AssemblyResolutionException: Failed to resolve assembly: 'System.ServiceModel.Duplex, Version=4.10.3.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'